libkpod/config.go

```diff
@@ -0,0 +1,63 @@
+package libkpod
+
+import (
+	"io/ioutil"
+
+	"github.com/BurntSushi/toml"
+	cstorage "github.com/containers/storage"
+)
+
+// Config stores the configuration options for libkpod
+type Config struct {
+	Root                     string
+	RunRoot                  string
+	Storage                  string
+	StorageOptions           []string
+	LogDir                   string
+	Runtime                  string
+	RuntimeUntrustedWorkload string
+	DefaultWorkloadTrust     string
+	Conmon                   string
+	ConmonEnv                []string
+	CgroupManager            string
+	DefaultTransport         string
+	InsecureRegistries       []string
+	SignaturePolicyPath      string
+}
+
+// FromFile populates the Config from the TOML-encoded file at the given path.
+// Returns errors encountered when reading or parsing the files, or nil
+// otherwise.
+func (c *Config) FromFile(path string) error {
```

Comment:
I didn't think we couldn't simply because the CRI-O config struct contains embedded structs, which requires the use of another struct.  The config is marshalled into the toml struct and converted into the config struct, whereas in libkpod we can marshal directly (at least for now).

See https://github.com/kubernetes-incubator/cri-o/blob/master/server/config.go#L186